lib/appear/lsof.rb

```diff
@@ -1,4 +1,5 @@
 require 'appear/service'
+require 'appear/memoizer'
```

Comment:
there's a ton of gems that do this already - is it worth using one of them instead of writing it yourself?